scripts/qc/alpha_second_time_report_generator.py

```diff
@@ -0,0 +1,89 @@
+import docx
+from docx.shared import Inches
+import os
+import glob
+from pdf2image import convert_from_path, convert_from_bytes
+from pdf2image.exceptions import (
+    PDFInfoNotInstalledError,
+    PDFPageCountError,
+    PDFSyntaxError
+)
+# Get the current working directory
+cwd = os.getcwd()
+output_file = input("This script assumes that there is already a .docx created \n if this is the first time runing this script please first run the alpha_first_time_report_generator.py \n Enter the name of the already existing output file: ")
```

Comment:
output_file = input("This script assumes that there is already a .docx created \n if this is the first time runing this script please first run the alpha_first_time_report_generator.py \n Enter the name of the already existing output file (do not include the .docx part): ")